### PR TITLE
Bug 2188502: Move SSH section to VM details on instancetype page

### PR DESCRIPTION
**Pull request by avivtur**
## 📝 Description

This PR is moving the SSH in the VM create with IT flow to live under a modal dialog instead the expandable section.

UX Doc: https://docs.google.com/document/d/1D1e8G0BmYbqokvJ7bLoo_uhvsMLATIhloQ-z2AwMiTY/edit# 
NOTE: Eventually this modal dialog will be applied across the plugin

## 🎥 Demo

Before:
ssh-section-b4

After:

https://github.com/kubevirt-ui/kubevirt-plugin/assets/67270715/4738b2bb-55ec-41e4-989e-0b23be193fd7

### CODE REVIEW

File: src/utils/components/SSHSecretSection/SSHSecretSection.tsx [hunk 1]
```diff
@@ -16,18 +18,27 @@ import HelpTextIcon from '../HelpTextIcon/HelpTextIcon';
...
     SecretSelectionOption.none,
   );
 
   const [secrets, ...loadedAndErrorData] = useK8sWatchResource<IoK8sApiCoreV1Secret[]>({
     groupVersionKind: modelToGroupVersionKind(SecretModel),
     isList: true,
-    ...(activeNamespace !== ALL_NAMESPACES_SESSION_KEY && {
-      namespace: activeNamespace,
+    ...(namespace !== ALL_NAMESPACES_SESSION_KEY && {
+      namespace: namespace,
```
Comment (metalice) [inline src/utils/components/SSHSecretSection/SSHSecretSection.tsx:41]: { namespace }

**Comment by yfrimanm** [comment]
+1 to placing the help text below the modal title.
Thanks @hstastna

**Pull request merged**